server/sonar-web/src/main/js/components/issue/BaseIssue.js

```diff
@@ -81,9 +93,20 @@ export default class BaseIssue extends React.PureComponent {
     document.removeEventListener('keypress', this.handleKeyPress);
   }
 
-  doIssueAction(action: string) {
-    this.issueView.$('.js-issue-' + action).click();
-  }
+  togglePopup = (popupName: string, open?: boolean) => {
```

Comment:
why do you allow `null` for `open` parameter? just to not write it?

I'd make it explicit, it'd simplify the code below